### DISCUSSION TITLE
Welcome to shapley_association_game Discussions!

### DISCUSSION THREAD

**Discussion by tomishima2904** [tomishima2904/shapley_association_game#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomishima2904** [comment]
現在はversion1での開発をしているため、開発用ブランチは`develop/1`となっています！
なので、最新のリポジトリは`develop/1`ブランチなので注意！
```
git pull origin develop/1:develop/1
```
↑のコマンドで`develop/1`ブランチの内容をローカルに反映させることができます。

**Reply by tomishima2904** [reply]
あと、この連想ゲーム関連のことで議論したい場合はなるべくこのリポジトリの[issues](https://github.com/tomishima2904/shapley_association_game/issues)もしくは[discussions](https://github.com/tomishima2904/shapley_association_game/discussions)に投げるようにお願いします！

**Comment by tomishima2904** [comment]
`.env`ファイルを更新したのでgoogle docsのやつを参照して変更してください

**Reply by tomishima2904** [reply]
2023/01/10 #54 で`.env`更新

**Comment by tomishima2904** [comment]
Docker周りを更新したら`git pull origin develop/1:develop/1`でプルした後で、`docker-compose build`してくれい

**Reply by tomishima2904** [reply]
2022/12/16 13:52 更新

**Comment by yuma283** [comment]
私が環境構築でつまづいた時に何をしたのかを記述しておく。
・Driveのdocsを参考に.envを更新する
・データベース側の準備を全て行う

**Comment by tomishima2904** [comment]
プルリクの時やコミット時にコメントを書くときに、#関連するissueの番号 を書いてくれるとレビューする時とか助かるかも

**Comment by tomishima2904** [comment]
バックもフロントも機能的にはほぼ完成に近いのであとはデザインを詰めるだけ！
一応`develop/1`ブランチのpullだけしておくとよい

**Comment by tomishima2904** [comment]
#51 こちらのプルリクで、フロントのメイン機能はほぼ完成した
あとはデザインの向上と機微な機能追加のみ

なので `develop/1` ブランチをプルしておくとよい

**Comment by tomishima2904** [comment]
【進捗状況】
なんやかんやでもうちょい頑張ればver.1がリリースできる状態まで仕上がってきました。残すは以下の作業のみ！
1. テンプレートデザインを詰める（`index.html`、`gaming.html`、`results.html`のみ、認証系はできたらでいい）。機能は完成しているのでjavascriptとか触る必要はほぼない、Tailwindいじるだけ。
1. テスト
1. AWSで本番環境構築（これは富島が請け負う）

なので、とりまデザインをいい感じにすればほぼ完成

認証系テンプレートの改変とか、更なる機能の盛り込みとかはver.2とかで趣味感覚でやるって感じで良いかな。

**Reply by tomishima2904** [reply]
【進捗状況】
アプリ完成したのであとはAWS上にデプロイするだけ
